src/utils/cookies.ts

```diff
@@ -1,9 +1,21 @@
+import Cookies from 'js-cookie';
+
+const NEXT_LOCALE = 'NEXT_LOCALE';
 const LOCALE_COOKIE_PERSISTENCE_PERIOD_MS = 86400000000000; // maximum milliseconds-since-the-epoch value https://stackoverflow.com/a/56980560/1931451
 
 // eslint-disable-next-line import/prefer-default-export
 export const setLocaleCookie = (newLocale: string) => {
   const date = new Date();
   date.setTime(LOCALE_COOKIE_PERSISTENCE_PERIOD_MS);
   // eslint-disable-next-line i18next/no-literal-string
-  document.cookie = `NEXT_LOCALE=${newLocale};expires=${date.toUTCString()};path=/`;
+  document.cookie = `${NEXT_LOCALE}=${newLocale};expires=${date.toUTCString()};path=/`;
+};
+
+export const getLocaleCookie = () => {
+  const value = Cookies.get(NEXT_LOCALE);
+  if (!value) {
+    return null;
+  }
+
+  return value;
```

Comment:
I don't think we need this one. We have been doing:
 `const { lang } = useTranslation();` 
 
 to get the current locale